src/lib/utils.ts

```diff
@@ -30,3 +30,37 @@ export const managementAppRequest = async (): Promise<ManagementAppResponse> =>
     const data = await response.json()
     return data
 }
+
+// Functions for interacting with the Trusted Output App
+export const toaRequest = async (): Promise<TOAResponse> => {
+    const endpoint = process.env.TOA_BASE_URL + '/api/runs' // `http://localhost:3002/api/runs`
```

Comment:
could probably default the `.env.example` `TOA_BASE_URL` to be `http://localhost:3002/api/runs` ?